packages/client/src/pages/index.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from "@emotion/styled";
+import Me from "components/me";
+import DocumentSelect from "components/select-document";
+import React from "react";
+import dynamic from "next/dynamic";
+
+const Editor = dynamic(() => import("../components/editor"), { ssr: false });
+
```

Comment:
Why the dynamic import with `ssr: false`?

Reply:
This prevents the page from being statically rendered. Read more about this feature of nextjs here https://nextjs.org/docs/advanced-features/dynamic-import